eis_toolkit/cli.py

```diff
@@ -3026,6 +3036,49 @@ def gamma_overlay_cli(input_rasters: INPUT_FILES_ARGUMENT, output_raster: OUTPUT
 # WOFE
 # TODO
 
+# --- TRAINING DATA TOOLS ---
+
+
+# BALANCE SMOTETOMEK
+@app.command()
+def balance_data_cli(
+    input_rasters: INPUT_FILES_ARGUMENT,
+    input_labels: INPUT_FILE_OPTION,
+    output_raster: OUTPUT_FILE_OPTION,
```

Comment:
I don't think we can return the stacked rasters as one. We have to return individual rasters that correspond to the input rasters, like in the `unify_rasters` CLI function. However, as I was digging into this, I learned that the balancing process changes the data size and spatial integrity of the data (idk if this is a concern). I think we need to think about this more and consult someone who knows about this method in this context.